drv/auxflash-api/src/lib.rs

```diff
@@ -51,7 +51,9 @@ pub enum AuxFlashError {
 #[repr(transparent)]
 pub struct AuxFlashId(pub [u8; 20]);
 
-#[derive(Copy, Clone, zerocopy::FromBytes, zerocopy::AsBytes)]
+#[derive(
```

Comment:
Nit: since we use `zerocopy::{AsBytes, FromBytes}` a lot in this file, maybe add them as an import (and let this stay a single line)

Reply:
fixed in dfe90f63